lisa/nic.py

```diff
@@ -257,7 +271,7 @@ def _get_node_nic_info(self) -> None:
             f"find any nics attached to {self._node.name}."
         ).is_greater_than(0)
 
-    def _get_default_nic(self) -> str:
+    def _get_default_nic(self) -> Tuple[str, str]:
```

Comment:
I don't prefer to return multiple values, it's hard to use. But if it's a short cut, please add comments to explain the meaning of each returned value.

Reply:
hmm returning multiple values to avoid executing same command on node again. Since we can get the default nic and default route both from the same command.
Will add explaination

Reply:
The mutiple value can be returned by an object, it's easier to use.

Reply:
Decided to not return anything, setting values in the method.